### ISSUE TITLE
DB API 2.0, INSERT problems

### ISSUE THREAD

**Issue by Barracuda72** [mymarilyn/clickhouse-driver#179]
**Describe the bug**
It seems there's no way to perform INSERT query when using DB API 2.0. (Or I'm dumb and can't figure that out).

**To Reproduce**

Preamble:

```
from clickhouse_driver import connect
conn = connect("clickhouse://localhost:9000/default")
cur = conn.cursor()
cur.execute('CREATE TABLE test (x Int32) ENGINE = Memory');
```

And now to the main thing.

This fails:

```
cur.execute('INSERT INTO test VALUES', [[1]])
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/.local/lib/python3.8/site-packages/clickhouse_driver/dbapi/cursor.py", line 112, in execute
    self._process_response(response)
  File "/home/user/.local/lib/python3.8/site-packages/clickhouse_driver/dbapi/cursor.py", line 315, in _process_response
    rows, columns_with_types = response
TypeError: cannot unpack non-iterable int object
```

But as you can see, it fails on the response parsing. The data is actually successfully inserted.

This fails in the exactly same way:

```
cur.execute('INSERT INTO test (x) VALUES', [{'x': 1}])
```

But the data is also successfully inserted.

And this two just hang waiting the response from the server:

```
cur.execute('INSERT INTO test (x) VALUES (%(x)d)', {'x': 1})
```

```
cur.execute('INSERT INTO test (x) VALUES', {'x': 1})
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/user/.local/lib/python3.8/site-packages/clickhouse_driver/dbapi/cursor.py", line 104, in execute
    response = execute(
  File "/home/user/.local/lib/python3.8/site-packages/clickhouse_driver/client.py", line 248, in execute
    rv = self.process_ordinary_query(
  File "/home/user/.local/lib/python3.8/site-packages/clickhouse_driver/client.py", line 446, in process_ordinary_query
    return self.receive_result(with_column_types=with_column_types,
  File "/home/user/.local/lib/python3.8/site-packages/clickhouse_driver/client.py", line 113, in receive_result
    return result.get_result()
  File "/home/user/.local/lib/python3.8/site-packages/clickhouse_driver/result.py", line 50, in get_result
    for packet in self.packet_generator:
  File "/home/user/.local/lib/python3.8/site-packages/clickhouse_driver/client.py", line 129, in packet_generator
    packet = self.receive_packet()
  File "/home/user/.local/lib/python3.8/site-packages/clickhouse_driver/client.py", line 143, in receive_packet
    packet = self.connection.receive_packet()
  File "/home/user/.local/lib/python3.8/site-packages/clickhouse_driver/connection.py", line 456, in receive_packet
    packet.type = packet_type = read_varint(self.fin)
  File "clickhouse_driver/varint.pyx", line 62, in clickhouse_driver.varint.read_varint
  File "clickhouse_driver/bufferedreader.pyx", line 55, in clickhouse_driver.bufferedreader.BufferedReader.read_one
  File "clickhouse_driver/bufferedreader.pyx", line 237, in clickhouse_driver.bufferedreader.BufferedSocketReader.read_into_buffer
KeyboardInterrupt
```

Data is not being inserted in both cases.

**Expected behavior**

I expect that at least one of this ways to perporm an INSERT should work.

**Versions**

OS: Ubuntu 20.04
clickhouse_driver version: 0.2.0
ClickHouse version: 18.16.1
Python version: 3.8.5

**Comment by xzkostyan** [comment]
Same issue https://github.com/mymarilyn/clickhouse-driver/issues/148#issuecomment-651907211

**Comment by Barracuda72** [comment]
> Same issue [#148 (comment)](https://github.com/mymarilyn/clickhouse-driver/issues/148#issuecomment-651907211)

I suppose not. In there @vladignatyev tried to insert many rows of data in one go, and I only need one. execute() should be perfectly fine for that.

Either way, I think that you should clearly state in documentation that your implementation of DB API 2.0 is WIP and for testing purposes only. It lacks more than it supports ATM.

**Comment by xzkostyan** [comment]
Fixed in 0.2.3.
